api/app/graphql/types/base_object.rb

```diff
@@ -3,5 +3,40 @@ class BaseObject < GraphQL::Schema::Object
     edge_type_class(Types::BaseEdge)
     connection_type_class(Types::BaseConnection)
     field_class Types::BaseField
+
+    def self.auto_field(*field_names)
+      field_names.each do |field_name|
+        raise("#{graphql_name} is not a model :(") unless ActiveRecord::Base.in?(klass.ancestors)
+
+        klass = graphql_name.constantize
+        column_metadata = klass.column_for_attribute(field_name)
+        field_type = field_type_from_column_metadata(column_metadata)
+
+        field field_name, field_type, null: column_metadata.null
```

Comment:
Typically we would say something like
```
field :top_score, Integer, null: false
```
directly in the GraphQL type. This is just a dynamic version of that.